cmd/cli/cancel/cancel.go

```diff
@@ -55,9 +58,13 @@ func newCancelCmd() *cobra.Command {
 		Long:    cancelLong,
 		Example: cancelExample,
 		Args:    cobra.ExactArgs(1),
-		PreRun:  applyPorcelainLogLevel,
+		PreRun:  handler.ApplyPorcelainLogLevel,
 		RunE: func(cmd *cobra.Command, cmdArgs []string) error {
-			return cancel(cmd, cmdArgs, cancelOptions)
+			err, exitcode := cancel(cmd, cmdArgs, cancelOptions)
+			if err != nil {
+				handler.Fatal(cmd, err, exitcode)
+			}
+			return nil
 		},
```

Comment:
since we are not returning errors, should we use `Run` instead of `RunE`?

Reply:
hum yeah good point, since we are calling `Fatal` from the root if there is an error it would probably be best to modify all command functions (`cancel`, `create`, `logs`, etc.) only return an error and handle the exit codes in a single place - the root command. wdyt?

Reply:
Makes sense. What happens if we return an error here? Why do need to handle the errors in root?

Reply:
So I thought it made sense too, but I was wrong, went with your suggestion in https://github.com/bacalhau-project/bacalhau/pull/2560/commits/20e66c7409e9230033a7c9b785aa5c5d0c79f5cc